src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -36,6 +38,9 @@ public void Start(DynamoModel model)
             if(service.GetTrackerFactory(GATrackerFactory.Name) == null)
                 service.Register(new GATrackerFactory(ANALYTICS_PROPERTY));
 
+            if (service.GetTrackerFactory(ADPTrackerFactory.Name) == null)
+                service.Register(new ADPTrackerFactory());
```

Comment:
can this fail for some reason?

Reply:
not really. At the moment...there is no more risk than the GATracker registration call.
We can wrap bot calls in a try catch...